### PR TITLE
added amgithubnotifier to promu and added docker build instructions

### PR DESCRIPTION
**Pull request by geekodour**
Signed-off-by: Hrishikesh Barman <hrishikeshbman@gmail.com>

### CODE REVIEW

**Comment by geekodour** [comment]
If I commit the changes in `git diff` it will fail in `make unused`. Seems like the tests in `Makefile.common` don't assume `go.mod` and `go.sum` in inner directories.

**Comment by krasi-georgiev** [comment]
some of there are done in https://github.com/prometheus/prombench/pull/240 so lets continue there.

**Comment by krasi-georgiev** [comment]
Aaah I didn't realise that this is for a different tool

File: tools/amGithubNotifier/README.md [hunk 1]
```diff
@@ -23,9 +23,16 @@ groups:
     annotations:
       description: 'description of the alert'
 ```
-
+#### Creating Docker images
+```
+docker build .
+```
+if building from repository root:
```
Comment (krasi-georgiev) [inline tools/amGithubNotifier/README.md:30]:
```suggestion
From the repository root:
```

**Comment by krasi-georgiev** [comment]
let me know when you fix the tests and will merge.

**Comment by geekodour** [comment]
After #253 I think this can be closed

**Comment by krasi-georgiev** [comment]
maybe leave the docker build note?

**Pull request closed without merging**